h/accounts/schemas.py

```diff
@@ -203,13 +203,15 @@ class SocialLoginSignupSchema(colander.Schema):
     comms_opt_in = comms_opt_in_node()
 
 
-class EmailChangeSchema(CSRFSchema):
-    email = email_node(title=_("Email address"))
```

Comment:
`title` args have been removed from all fields as these schemas are no longer used for rendering forms.